pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarConnectorFactory.java

```diff
@@ -49,7 +49,7 @@ public ConnectorHandleResolver getHandleResolver() {
     @Override
     public Connector create(String connectorId, Map<String, String> config, ConnectorContext context) {
         requireNonNull(config, "requiredConfig is null");
-        log.debug("Creating Pulsar connector with configs: %s", config);
+        log.info("Creating Pulsar connector with configs: %s", config);
```

Comment:
Do we need to change this to info level?  Presto will automatically log the connector configs